source/01-LE/02.ptx

```diff
@@ -240,23 +389,33 @@
 <activity estimated-time='5'>
     <statement>
     <p>
-We can rewrite the previous in terms of  equivalences of augmented matrices
+Here we've written the sorted linear systems from <xref ref="LE2-activity-sort"/>
+as augmented matrices.
     </p>
     <md alignment="alignat" alignat-columns='3'>
         <mrow>
-        \left[\begin{array}{ccc|c} 2 &amp; 5 &amp; 3 &amp; 7 \\ -1 &amp; -1 &amp; 1 &amp; 1 \\ 1 &amp; 2 &amp; 1 &amp; 3 \end{array}\right] &amp; \sim &amp;
-        \left[\begin{array}{ccc|c} \markedPivot{1} &amp; 2 &amp; 1 &amp; 3 \\ -1 &amp; -1 &amp; 1 &amp; 1 \\ 2 &amp; 5 &amp;  3 &amp; 7  \end{array}\right] &amp; \sim &amp;
-        \left[\begin{array}{ccc|c} \markedPivot{1} &amp; 2 &amp; 1 &amp; 3 \\ 0 &amp; 1 &amp; 2 &amp; 4 \\ 2 &amp; 5 &amp; 3 &amp; 7  \end{array}\right] \sim 
+\left[\begin{array}{ccc|c} 2 &amp; 5 &amp; 3 &amp; 7 \\ -1 &amp; -1 &amp; 1 &amp; 1 \\ 1 &amp; 2 &amp; 1 &amp; 3 \end{array}\right] &amp; \sim &amp;
+\left[\begin{array}{ccc|c} \markedPivot{1} &amp; 2 &amp; 1 &amp; 3 \\ -1 &amp; -1 &amp; 1 &amp; 1 \\ 2 &amp; 5 &amp;  3 &amp; 7  \end{array}\right] &amp; \sim &amp;
+\left[\begin{array}{ccc|c} \markedPivot{1} &amp; 2 &amp; 1 &amp; 3 \\ 0 &amp; 1 &amp; 2 &amp; 4 \\ 2 &amp; 5 &amp; 3 &amp; 7  \end{array}\right]\sim 
         </mrow>
         <mrow>
-        \left[\begin{array}{ccc|c} \markedPivot{1} &amp; 2 &amp; 1 &amp; 3 \\ 0 &amp; \markedPivot{1} &amp; 2 &amp; 4 \\ 0 &amp; 1 &amp; 1 &amp; 1  \end{array}\right] &amp; \sim &amp;
-        \left[\begin{array}{ccc|c} \markedPivot{1} &amp; 0 &amp; -1 &amp; 1 \\ 0 &amp; \markedPivot{1} &amp; 2 &amp; 4 \\ 0 &amp; 1 &amp; 1 &amp; 1  \end{array}\right] &amp; \sim &amp;
-        \left[\begin{array}{ccc|c} \markedPivot{1} &amp; 0 &amp; -1 &amp; 1 \\ 0 &amp; \markedPivot{1} &amp; 1 &amp; 1 \\ 0 &amp; 0 &amp; -1 &amp; -3  \end{array}\right] 
+\sim \left[\begin{array}{ccc|c} \markedPivot{1} &amp; 2 &amp; 1 &amp; 3 \\ 0 &amp; \markedPivot{1} &amp; 2 &amp; 4 \\ 0 &amp; 1 &amp; 1 &amp; 1  \end{array}\right] &amp; \sim &amp;
+\left[\begin{array}{ccc|c} \markedPivot{1} &amp; 0 &amp; -1 &amp; 1 \\ 0 &amp; \markedPivot{1} &amp; 2 &amp; 4 \\ 0 &amp; 1 &amp; 1 &amp; 1  \end{array}\right] &amp; \sim &amp;
+\left[\begin{array}{ccc|c} \markedPivot{1} &amp; 0 &amp; -1 &amp; 1 \\ 0 &amp; \markedPivot{1} &amp; 1 &amp; 1 \\ 0 &amp; 0 &amp; -1 &amp; -3  \end{array}\right] 
         </mrow>
      </md>
 <p>
-        Determine the row operation(s) necessary in each step to transform the most complicated
-    system's augmented matrix into the simplest.
+Assign the following row operations to each step used to manipulate each
```

Comment:
Give choices for row operations to narrow student choices and reduce overhead to inquiry.